packages/amplication-client/src/Resource/create-resource/wizard-pages/ImgSvg.tsx

```diff
@@ -1,9 +1,34 @@
+import React, { useEffect, useState } from "react";
 import classNames from "classnames";
 
-const ImgSvg = (
+export const useImageLoader = (currentSrc: string) => {
+  const [imageSrc, _setImageSrc] = useState(null);
```

Comment:
What does `_setImageSrc` mean?

Reply:
its a set function of useState. sometime it's recommended to add _ before set

Reply:
But we are not using it anywhere. It's not our convention 🙈